spacy/errors.py

```diff
@@ -843,6 +843,10 @@ class Errors:
              "DependencyMatcher token patterns. The token pattern in "
              "RIGHT_ATTR should return matches that are each exactly one token "
              "long. Invalid pattern:\n{node}")
+    E1017 = ("Heads passed without deps. Normally this is a mistake, but if "
+             "intentional then pass filler deps such as `[\"dep\"] * len(heads)`.")
```

Comment:
I think it would be better to give a little more explanation and not assume that an unlabeled dependency parse is a mistake, something like:

A Doc object requires both 'deps' and 'heads' for dependency parses. If no dependency labels are available, provide placeholder deps such as `deps=[\"dep\"]*len(heads)`.

Reply:
That sounds good, updated the message.